semantic_conventions.md

```diff
@@ -55,6 +57,15 @@ The following Span tags combine to model RPCs:
 - `error`: whether the RPC ended in an error
 - `peer.hostname`, `peer.ipv4`, `peer.ipv6`, `peer.port`, `peer.service`: optional tags that describe the RPC peer (often in ways it cannot assess internally)
 
+### Message Oriented Middleware
+
+The following Span tags combine to model Message Oriented Middleware communications:
+
+- `mom.destination` and `mom.type`: as described in the table above
```

Comment:
Can you give more details about how to set `mom.destination`? e.g. when the client sends a message for kafka.